src/components/nav-user/nav-user.tsx

```diff
@@ -21,19 +21,19 @@ export class CalciteNavUser {
   //
   //--------------------------------------------------------------------------
   /** When `true`, visually highlight the component */
-  @Prop({ reflect: true }) active?;
+  @Prop({ reflect: true }) active: boolean;
 
   /** Specifies the `src` to an image to display in the Avatar (remember to add a token if the user is private). */
-  @Prop({ reflect: true }) thumbnail?;
+  @Prop({ reflect: true }) thumbnail: string;
 
   /** Specifies the subtext to display, for example a user organization or role */
-  @Prop({ reflect: true }) username?: string;
+  @Prop({ reflect: true }) username: string;
 
   /** Specifies the text to display, for example a user name or full name */
-  @Prop({ reflect: true }) fullName?: string;
+  @Prop({ reflect: true }) fullName: string;
 
-  /** When `true`, makes `fullName` and `username` visible */
-  @Prop({ reflect: true }) textEnabled?: boolean;
```

Comment:
I think we want to leave this for `nav-user` and `nav-logo`.

Reply:
restored.